plugins/formal-verification/formver.plugin/src/org/jetbrains/kotlin/formver/reporting/VerifierErrorInterpreter.kt

```diff
@@ -81,4 +84,14 @@ class VerifierErrorInterpreter {
         is ConsistencyError -> reportConsistencyError(source, error, context)
         is VerificationError -> reportVerificationError(source, error, errorStyle, context)
     }
+
+    private val EventOccurrencesRange.asUserFriendlyMessage: String
+        get() = when (this) {
+            EventOccurrencesRange.ZERO -> "zero"
```

Comment:
I think the grammar here is incorrect, same for unknown.

Reply:
Mhh maybe for `ZERO` I could do:

```
Could not verify that function '{0}' is never called.
```

But I don't know for `UNKNOWN`. Any suggestions?

Reply:
Can this error ever arise for a `callsInPlace` contract with occurrences range unknown?

Reply:
I think not. I will ignore that case then

Reply:
After watching the [documentation](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.contracts/-invocation-kind/), it seems that ZERO and UNKNOWN are never exposed to the user.